src/server/scripts/poller.js

```diff
@@ -103,13 +110,14 @@ export const pollRepositories = (checker) => {
           }
         } catch (e) {
           raven_client.captureException(e);
-          logger.error(`${owner}/${name}: FAILED (${e.message})`);
+          logger.error(`${owner}/${name}: FAILED (${e.message || e})`);
         }
         logger.info('==========');
       });
 
       locked_promises.push(p);
     });
+    if (locked_promises.length == 0) { return []; }
```

Comment:
Is this needed?

If `locked_promises` is empty `Promise.all(locked_promises)` would return automatically resolved promise, which is expected I think? Right now it seems that we return promise or empty array.

Reply:
IF block dropped, it doesn't help with the problem we are still seeing with knex + postgres when there are no repositories registered.